metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestPoliciesStep.java

```diff
@@ -88,11 +98,45 @@ public void execute() throws IOException, URISyntaxException {
     log.info("Successfully ingested default access policies.");
   }
 
+  /**
+   * Send MCLs for each policy to refill the policy search index
+   */
+  private void sendMCL() throws URISyntaxException {
+    log.info("Pushing MCLs for all policies");
```

Comment:
what policies? bootstrap policies? do they already exist inside the document store?

Reply:
Leaving for record. The current policy index is empty bc there are no searchable fields. So this step makes sure we reingest the existing policies so they get in the index one time